src/settings.js

```diff
@@ -1,7 +1,7 @@
 export default {
   account: {
-    pub: 'ak_2bKhoFWgQ9os4x8CaeDTHZRGzUcSwcXYUrM12gZHKTdyreGRgG',
-    priv: '0eb8cefe04593f2960ddb7d731321b709cec5ce10625334542e410630f0b02d4d1a124ce191ef08e8d2d8fbf424504013c9b8aaecec6f71785f2ee20ddf3a688'
+    pub: 'ak_2mwRmUeYmfuW93ti9HMSUJzCk1EYcQEfikVSzgo6k2VghsWhgU',
```

Comment:
please don't update the account here, because the one provided is not generalized